docs/pulseload.md

```diff
@@ -62,3 +62,60 @@ ex: <https://community-tc.services.mozilla.com/pulse-messages/>
 
 [pulse guardian]: https://pulseguardian.mozilla.org/whats_pulse
 [yml schema]: https://github.com/mozilla/treeherder/blob/master/schemas/pulse-job.yml
+
+**Here is an example flow you can use:**
+
+## Setup
+
+(These steps should help you set up everything you need to ingest data locally while working on perfherder)
+
+**Create a pulse guardian account and run:**
+
+```bash
+export PULSE_URL=amqp://USER:PASSWORD@pulse.mozilla.org:5671/?ssl=1
+yarn install
+docker-compose up --build
+```
+
+**Run each of these commands in multiple windows:**
+
+```bash
+docker-compose up --build
+docker-compose run -e PROJECTS_TO_INGEST=autoland backend celery -A treeherder worker --concurrency 1
+```
+
+---
+
+**Run the db viewer**
+
+You can use **dbeaver-ce**
+
+**Connect to the following while `docker-compose up --build` from the previous step is running:**
+
+`Serverhost: localhost`
+`Port: 3306`
+`Database: treeherder`
+`Username: root`
+`No password`
+
+---
+
+##### Run the following in separate window while running above to do ingestion
```

Comment:
Below this, we should add a comment saying that these commands actually perform the fetches for the data, and that they are run sequentially in the same window. The first command makes the second one run faster. Also, we should mention here the `--enable-eager-celery` triggers the log parsing which is required to capture the `PERFHERDER_DATA` output.